backend/src/modules/tournament-tables/core/application/command/AssignTournamentTables.ts

```diff
@@ -1,8 +1,8 @@
 export class AssignTournamentTables {
   readonly tournamentId: string;
-  readonly tables: { tableNumber: number; tableName: string }[];
+  readonly tables: { tableNumber: number; tableName: string; availableToPlay?: boolean }[];
```

Comment:
Do czego nam to tutaj potrzebne :) ? Chcemy od razu dać stoły jako niedostepne?

Reply:
Zostawiłam taką opcję :)